modules/ROOT/pages/automated-policies-landing-page.adoc

```diff
@@ -0,0 +1,22 @@
+= Automated Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Automated policies make it easy for your API management team to satisfy compliance in common security, and logging requirements by automatically applying the same set of provided policies to all of your configured deployed runtimes. +
+Your API development team can focus on building API proxies without worrying about compliance requirements because all of your deployed API proxies would inherit the configured automated policy for their runtime versions.
+
+Automated policies ensure consistency and reduce possible errors because an administrator no longer needs to remember to manually insert a policy to enforce common standards.
+
+All of the xref:policies-ootb-landing-page.adoc[Mule 4 Provided Policies] can be configured as automated policies. +
```

Comment:
Custom policies could be Automated Policies too

Reply:
Made some changes here: 7a7f996